### PR TITLE
Update test plan with desktop resize tests

### PR DESCRIPTION
**Pull request by ibeckermayer**

### CODE REVIEW

**Comment by ibeckermayer** [comment]
> What about session recordings that have this message in them?

I knew there was a reason we didn't do this before... reverting and will add a comment so that I don't keep doing this.

File: .github/ISSUE_TEMPLATE/testplan.md [hunk 1]
```diff
@@ -1292,12 +1292,18 @@ tsh bench web sessions --max=5000 --web user ls
 - Warnings/Errors (test by applying [this patch](https://gist.github.com/ibeckermayer/7591333275e87ad0d7afa028a7bb54cb))
   - [ ] Induce the backend to send a TDP Notification of severity warning (1), confirm that a warning is logged in the warning dropdown
   - [ ] Induce the backend to send a TDP Notification of severity error (2), confirm that session is terminated and error popup is shown
-  - [ ] Induce the backend to send a TDP Error, confirm that session is terminated and error popup is shown (confirms backwards compatibility w/ older w_d_s starting in Teleport 12)
```
Comment (zmb3) [inline .github/ISSUE_TEMPLATE/testplan.md:1295]: Why remove this?

Comment (ibeckermayer) [reply .github/ISSUE_TEMPLATE/testplan.md:1295]: My initial thinking was that we no longer actually send any `tdp.Error`s, they've been replaced with tdp.Notification (same reason why I'd ripped out `tdp.Error` generally, before you pointed out the recording problem). I can change this to confirming that a recording with this works.

Comment (ibeckermayer) [reply .github/ISSUE_TEMPLATE/testplan.md:1295]: 064ec4f34463d64db4e85283652d7cc1380947af

**Review by probakowski — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/testplan.md [hunk 2]
```diff
@@ -1292,12 +1292,20 @@ tsh bench web sessions --max=5000 --web user ls
...
+  - [ ] Induce the backend to send a TDP Error, confirm that session is terminated and error popup is shown. Confirm that the error is
+        shown at the end of the playback of this session (confirms backwards compatibility w/ recordings from older w_d_s pre Teleport 12).
```
Comment (zmb3) [inline .github/ISSUE_TEMPLATE/testplan.md:1296]: If we actually want to test this, we should stash an old recording somewhere. No one is going to want to stand up Teleport 12 and set up desktop access just to get a recording to test.

**Review by zmb3 — APPROVED** [review]
I'd update the PR description, squash the commits, and make a more accurate commit message prior to merging.

**Review by russjones — APPROVED** [review]

**Pull request merged**